### ISSUE TITLE
Is it possible to copy the file(s) listed in "source"?

### ISSUE THREAD

**Issue by KaptenJansson** [blittle/bower-installer#124]
My setup is basically like this:

bower.json
{
  "name": "blah",
  "dependencies": {
    "polymer": "Polymer/polymer#*",
    "requireJS": latest
  },
  "install": {
    "path": "src/requirejs",
    "sources": {
      "requireJS": "bower_components/requireJS/require.js"
    },
}

* Expected result when I run bower-installer:
requireJS is copied to src/requirejs
polymer is NOT copied at all, just installed in the default bower or .bowerrc directory.

* Actual result when I run bower-installer:
polymer and requireJS is copied to src/requirejs, in other words all dependencies are copied to the path location, not just file(s) in "sources"

I cannot use ignore due to I do not know all the dependencies of the packages in "dependencies". My example only has two dependencies but my project has tons more hence using ignore does not scale.

Basically I want to install individual dependencies/packages explicitly (install by file extension will obviously not work here).

**Comment by donnes** [comment]
I need it too

**Comment by donnes** [comment]
I have an temporary solution.

Use this fork for your ```bower-installer```: https://github.com/Gerhard-Kanzler/bower-installer

Now, see my bower.json example:

```
"install": {
    "path": {
      "css": "public/assets/css",
      "js": "public/assets/js",
      "eot": "public/assets/fonts/",
      "svg": "public/assets/fonts/",
      "ttf": "public/assets/fonts/",
      "woff": "public/assets/fonts/",
      "woff2": "public/assets/fonts/"
    },
    "sources": {
      "jquery": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/jquery/dist/jquery.min.js": "jquery.min.js"
          }
        ]
      },
      "angular": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/angular/angular.min.js": "angular.min.js"
          }
        ]
      },
      "angular-ui-router": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/angular-ui-router/release/angular-ui-router.min.js": "angular-ui-router.min.js"
          }
        ]
      },
      "angular-bootstrap": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/angular-bootstrap/ui-bootstrap-tpls.min.js": "ui-bootstrap-tpls.min.js"
          }
        ]
      },
      "bootstrap": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/bootstrap/dist/js/bootstrap.min.js": "bootstrap.min.js"
          },
          {
            "bower_components/bootstrap/dist/css/bootstrap.min.css": "bootstrap.min.css"
          }
        ]
      },
      "font-awesome": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/font-awesome/css/font-awesome.min.css": "font-awesome.min.css"
          },
          {
            "bower_components/font-awesome/fonts/fontawesome-webfont.eot": "fontawesome-webfont.eot"
          },
          {
            "bower_components/font-awesome/fonts/fontawesome-webfont.svg": "fontawesome-webfont.svg"
          },
          {
            "bower_components/font-awesome/fonts/fontawesome-webfont.ttf": "../fonts/fontawesome-webfont.ttf"
          },
          {
            "bower_components/font-awesome/fonts/fontawesome-webfont.woff": "../fonts/fontawesome-webfont.woff"
          },
          {
            "bower_components/font-awesome/fonts/fontawesome-webfont.woff2": "../fonts/fontawesome-webfont.woff2"
          }
        ]
      },
      "owl.carousel": {
        "subDir": false,
        "mapping": [
          {
            "bower_components/owl.carousel/dist/owl.carousel.min.js": "owl.carousel.min.js"
          },
          {
            "bower_components/owl.carousel/dist/assets/owl.carousel.min.css": "owl.carousel.min.css"
          }
        ]
      }
    }
  }
```

**Comment by chernosliv** [comment]
did you try ignore? But it looks like it's per dependency only as in my case I cannot ignore individual files.
From documentation:
```
{
  "name" : "test",
  "version": "0.1",
  "dependencies": {
    "ember-model" : "0.0.8"
  },
  "install": {
    "path": "build/src",
    "ignore": ["ember", "handlebars"]
  }
}
```

**Comment by KaptenJansson** [comment]
Sorry for missing the responses and thanks for responding ;)

I've not dealt with this for a long while but will take another look soonish and see if this is still needed and try out your suggestions!
